### PR TITLE
Replace Lazy mode with torch.compile mode in Readme

### PR DESCRIPTION
**Pull request by MohitIntel**

### CODE REVIEW

File: examples/language-modeling/README.md [hunk 1]
```diff
@@ -157,15 +157,15 @@ This example has been validated with the following DeepSpeed ZeRO-2 config: http
...
 Following the RoBERTa paper, we use dynamic masking rather than static masking. The model may, therefore,
 converge slightly slower (over-fitting takes more epochs).
 
 
 ### Single-card Training
 
 ```bash
-python run_mlm.py \
+PT_HPU_LAZY_MODE=0 PT_ENABLE_INT64_SUPPORT=1 python run_mlm.py \
     --model_name_or_path roberta-base \
```
Comment (abala-unicorn) [inline examples/language-modeling/README.md:169]: Roberta Base for now is supported in lazy mode. Roberta Large to be supported in torch.compile

File: examples/language-modeling/README.md [hunk 2]
```diff
@@ -175,17 +175,19 @@ python run_mlm.py \
     --do_eval \
     --output_dir /tmp/test-mlm \
     --use_habana \
-    --use_lazy_mode \
+    --use_lazy_mode False \
+    --torch_compile \
+    --torch_compile_backend hpu_backend \
     --use_hpu_graphs_for_inference \
     --gaudi_config_name Habana/roberta-base \
     --throughput_warmup_steps 3 \
```
Comment (abala-unicorn) [inline examples/language-modeling/README.md:183]: Roberta Base for now is supported in lazy mode. Roberta Large to be supported in torch.compile

File: examples/language-modeling/README.md [hunk 3]
```diff
@@ -195,7 +197,9 @@ python run_mlm.py \
     --do_eval \
     --output_dir /tmp/test-mlm \
     --use_habana \
-    --use_lazy_mode \
+    --use_lazy_mode False \
+    --torch_compile \
+    --torch_compile_backend hpu_backend \
     --use_hpu_graphs_for_inference \
     --gaudi_config_name Habana/roberta-base \
```
Comment (abala-unicorn) [inline examples/language-modeling/README.md:204]: Roberta Base for now is supported in lazy mode, Roberta Large to be supported in torch.compile

**Review by abala-unicorn — CHANGES_REQUESTED** [review]

File: examples/language-modeling/README.md (hunk 3 above)
Comment (MohitIntel) [reply examples/language-modeling/README.md]: Fixed in new commit "Changes after review". Rebased to latest habana-main.

File: examples/language-modeling/README.md (hunk 2 above)
Comment (MohitIntel) [reply examples/language-modeling/README.md]: Fixed in new commit "Changes after review". Rebased to latest habana-main.

File: examples/language-modeling/README.md (hunk 1 above)
Comment (MohitIntel) [reply examples/language-modeling/README.md]: Fixed in new commit "Changes after review". Rebased to latest habana-main.

File: examples/language-modeling/README.md [hunk 4]
```diff
@@ -228,7 +228,9 @@ python ../gaudi_spawn.py \
     --throughput_warmup_steps 3 \
     --bf16
 ```
-
+**Note:** To use the RoBERTa-Large model in eager mode with torch.compile enabled,
```
Comment (libinta) [inline examples/language-modeling/README.md:231]: Can you change to 
Roberta-large is enabled with torch.compile with the following cmd change

File: examples/question-answering/README.md [hunk 5]
```diff
@@ -154,6 +152,10 @@ python run_qa.py \
   --bf16
 ```
 
+**Note:** To use eager mode with torch.compile enabled in the above commands,  
+a) Set the following environment variables `PT_HPU_LAZY_MODE=0` and `PT_ENABLE_INT64_SUPPORT=1`.
```
Comment (libinta) [inline examples/question-answering/README.md:156]: which model?

File: examples/language-modeling/README.md [hunk 6]
```diff
@@ -229,6 +229,11 @@ python ../gaudi_spawn.py \
     --bf16
 ```
 
+## Training in torch.compile mode
```
[the remark this answers is not in the document]
Comment (MohitIntel) [reply examples/language-modeling/README.md]: Fixed in latest commit

**Review by jiminha — APPROVED** [review]

**Review by ssarkar2 — APPROVED** [review]

**Pull request merged**